2025-03-12 11:23:48,630 INFO    MainThread:4496 [wandb_setup.py:_flush():68] Current SDK version is 0.19.2
2025-03-12 11:23:48,630 INFO    MainThread:4496 [wandb_setup.py:_flush():68] Configure stats pid to 4496
2025-03-12 11:23:48,630 INFO    MainThread:4496 [wandb_setup.py:_flush():68] Loading settings from C:\Users\hyunoh\.config\wandb\settings
2025-03-12 11:23:48,630 INFO    MainThread:4496 [wandb_setup.py:_flush():68] Loading settings from C:\Users\hyunoh\Documents\Codes\BMD_code\wandb\settings
2025-03-12 11:23:48,630 INFO    MainThread:4496 [wandb_setup.py:_flush():68] Loading settings from environment variables
2025-03-12 11:23:48,630 INFO    MainThread:4496 [wandb_init.py:_log_setup():523] Logging user logs to C:\Users\hyunoh\Documents\Codes\BMD_code\wandb\run-20250312_112348-wv8tb4bx\logs\debug.log
2025-03-12 11:23:48,630 INFO    MainThread:4496 [wandb_init.py:_log_setup():524] Logging internal logs to C:\Users\hyunoh\Documents\Codes\BMD_code\wandb\run-20250312_112348-wv8tb4bx\logs\debug-internal.log
2025-03-12 11:23:48,630 INFO    MainThread:4496 [wandb_init.py:init():641] calling init triggers
2025-03-12 11:23:48,631 INFO    MainThread:4496 [wandb_init.py:init():647] wandb.init called with sweep_config: {}
config: {'learning_rate': 5e-06, 'epochs': 150, 'batch_size': 1, 'optimizer': 'AdamW', 'scheduler': 'CosineAnnealingLR'}
2025-03-12 11:23:48,631 INFO    MainThread:4496 [wandb_init.py:init():674] starting backend
2025-03-12 11:23:48,828 WARNING MainThread:4496 [wandb_init.py:init():1293] interrupted
Traceback (most recent call last):
  File "C:\Users\hyunoh\anaconda3\envs\spine\lib\site-packages\wandb\sdk\wandb_init.py", line 1289, in init
    return wi.init()
  File "C:\Users\hyunoh\anaconda3\envs\spine\lib\site-packages\wandb\sdk\wandb_init.py", line 677, in init
    service = self._wl.ensure_service()
  File "C:\Users\hyunoh\anaconda3\envs\spine\lib\site-packages\wandb\sdk\wandb_setup.py", line 280, in ensure_service
    self._connection = service_connection.connect_to_service(self._settings)
  File "C:\Users\hyunoh\anaconda3\envs\spine\lib\site-packages\wandb\sdk\lib\service_connection.py", line 40, in connect_to_service
    return _start_and_connect_service(settings)
  File "C:\Users\hyunoh\anaconda3\envs\spine\lib\site-packages\wandb\sdk\lib\service_connection.py", line 75, in _start_and_connect_service
    proc.start()
  File "C:\Users\hyunoh\anaconda3\envs\spine\lib\site-packages\wandb\sdk\service\service.py", line 232, in start
    self._launch_server()
  File "C:\Users\hyunoh\anaconda3\envs\spine\lib\site-packages\wandb\sdk\service\service.py", line 224, in _launch_server
    self._wait_for_ports(fname, proc=internal_proc)
  File "C:\Users\hyunoh\anaconda3\envs\spine\lib\site-packages\wandb\sdk\service\service.py", line 105, in _wait_for_ports
    time.sleep(0.2)
KeyboardInterrupt
